content/agent/kubernetes/helm.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Deploying Datadog in Kubernetes using Helm 
+kind: documentation
+further_reading:
+- link: "agent/kubernetes/daemonset_setup"
+  tag: "documentation"
+  text: "Kubernetes DaemonSet Setup"
+- link: "agent/kubernetes/host_setup"
+  tag: "documentation"
+  text: "Kubernetes Host Setup"
+- link: "agent/kubernetes/metrics"
+  tag: "documentation"
+  text: "Kubernetes Metrics"
+---
+
+Helm is a package management tool for Kubernetes. 
+
+## Installing Helm
+
+### Installing the Helm client
+
+{{< tabs >}}
+{{% tab "macOS (Homebrew)" %}}
+
+```bash
+brew install kubernetes-helm
+```
+
+{{% /tab %}}
+{{% tab "Linux (Snap)" %}}
+
+```bash
+sudo snap install helm --classic
+```
+
+{{% /tab %}}
+{{% tab "Windows (Chocolatey)" %}}
+
+```bash
+choco install kubernetes-helm
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+For other platforms and methods of installing Helm, refer to the [Helm documentation][1].
+
+### Installing the Helm server (Tiller)
+
+If your Kubernetes environment does not use RBAC, the following command installs Tiller in your cluster:
+
+```bash
+helm init
+```
+
+Refer to [Helm's Tiller documentation][2] for further details.
+
+If your Kubernetes cluster is RBAC-enabled, use the following RBAC to deploy Tiller.
+
+```yaml
+apiVersion: v1  
+kind: ServiceAccount  
+metadata:  
+ name: tiller  
+ namespace: kube-system  
+---  
+apiVersion: rbac.authorization.k8s.io/v1beta1  
+kind: ClusterRoleBinding  
+metadata:  
+ name: tiller  
+roleRef:  
+ apiGroup: rbac.authorization.k8s.io  
+ kind: ClusterRole  
+ name: cluster-admin  
+subjects:  
+ - kind: ServiceAccount  
+   name: tiller  
+   namespace: kube-system
+```
+
+With this as your `tiller-rbac-config.yaml`, then run:
+
+```bash
+kubectl create -f tiller-rbac-config.yaml
+
+helm init --service-account tiller
+```
+
+Refer to [Helm's Tiller/RBAC documentation][3] for further details.
+
+### Verify installation
+
+To verify your installation, run:
+
+```bash
+kubectl get pods -n kube-system
```

Comment:
nit: the namespace can be crowded on hosted k8s clusters, we can narrow down the query by label matching, for example:

>  kubectl get pod -n kube-system -l app=helm
NAME                            READY   STATUS    RESTARTS   AGE
tiller-deploy-f54b67464-jl5gm   1/1     Running   0          3h16m